src/renderer/components/ScreenSharePicker.tsx

```diff
@@ -88,10 +88,10 @@ addPatch({
             bitrateMax: 8000000,
             bitrateTarget: 600000
         });
-        Object.assign(opts.encode, {
-            framerate,
-            pixelCount: height * width
-        });
+//         Object.assign(opts.encode, {
```

Comment:
```suggestion
        if (opts?.encode)
          Object.assign(opts.encode, {
            framerate,
            pixelCount: height * width
          });
```

Reply:
fixed this in [006f4e9](https://github.com/Vencord/Vesktop/pull/489/commits/006f4e96dea21fee018e446e84a19a92ddabbd1e)